README.md

```diff
@@ -46,6 +50,8 @@ A collection of the best CSS resources.
 - [CSS Battle](https://cssbattle.dev/)
 - [Flexbox Defense](http://www.flexboxdefense.com/)
 - [Flexbox](https://mastery.games/flexboxzombies/)
+- [Knights of the Flexbox Table](https://knightsoftheflexboxtable.com/)
+- [Grid attackpermalink](https://codingfantasy.com/games/css-grid-attack/play)
```

Comment:
Looks to be a typo here for "Grid Attack", where it includes "permalink".

Reply:
updated it.